codegenerator/cli/templates/static/codegen/src/db/Migrations.res

```diff
@@ -93,6 +93,20 @@ module EntityHistory = {
           -- Convert the record into a JSONB object, iterate over each field
           SELECT jsonb_object_agg(d.key, 
                                   CASE 
+                                      WHEN jsonb_typeof(d.value) = 'array' THEN 
+                                          CASE 
+                                              WHEN jsonb_array_length(d.value) = 0 THEN '[]'::JSONB  -- Return empty array
+                                              ELSE 
+                                                  (
+                                                      SELECT jsonb_agg(
+                                                          CASE 
+                                                              WHEN jsonb_typeof(elem) = 'number' THEN to_jsonb(elem::TEXT)
+                                                              ELSE elem
+                                                          END
+                                                      )
+                                                      FROM jsonb_array_elements(d.value) AS elem
+                                                  )
+                                          END
```

Comment:
During testing I realised I wasn't handling array types 😓

For now while have a union type query for the diff we need a way to represent the entity data as json. So I'm ok with this for an initial implementation but it's far from ideal.